C/CUTEst/build_tarballs.jl

```diff
@@ -81,14 +91,14 @@ all_load="--whole-archive"
 noall_load="--no-whole-archive"
 extra=""
 if [[ "${target}" == *-apple-* ]]; then
-    all_load="-all_load"
-    noall_load="-noall_load"
-    extra="-Wl,-undefined -Wl,dynamic_lookup -headerpad_max_install_names"
+  all_load="-all_load"
+  noall_load="-noall_load"
+  extra="-Wl,-undefined -Wl,dynamic_lookup -headerpad_max_install_names"
 fi
-cd $CUTEST/objects/$MYARCH/double
-gfortran -fPIC -shared ${extra} -Wl,${all_load} libcutest.a -Wl,${noall_load} -o libcutest_double.${dlext}
 cd $CUTEST/objects/$MYARCH/single
-gfortran -fPIC -shared ${extra} -Wl,${all_load} libcutest.a -Wl,${noall_load} -o libcutest_single.${dlext}
+gfortran -fPIC -shared ${extra} -Wl,${all_load} libcutest.a -Wl,${noall_load} -o ${libdir}/libcutest_single.${dlext}
+cd $CUTEST/objects/$MYARCH/double
+gfortran -fPIC -shared ${extra} -Wl,${all_load} libcutest.a -Wl,${noall_load} -o ${libdir}/libcutest_double.${dlext}
```

Comment:
```suggestion
gfortran -fPIC -shared ${extra} $(flagon -Wl,--whole-archive) libcutest.a $(flagon -Wl,--no-whole-archive) -o ${libdir}/libcutest_single.${dlext}
cd $CUTEST/objects/$MYARCH/double
gfortran -fPIC -shared ${extra} $(flagon -Wl,--whole-archive) libcutest.a $(flagon -Wl,--no-whole-archive) -o ${libdir}/libcutest_double.${dlext}
```
and remove all the settings of `all_load` above.